lib/features/support/env.rb

```diff
@@ -4,4 +4,4 @@
 $api_key = SecureRandom.hex(16).tr('+/=', 'xyz')
 
 # A regex providing the pattern expected from timestamps
-TIMESTAMP_REGEX = /^\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:[\d\.]+Z?$/
+TIMESTAMP_REGEX = /^\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:[\d\.]+(Z|[+-]\d{2}:\d{2})?$/
```

Comment:
```suggestion
TIMESTAMP_REGEX = /^\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:[\d\.]+(Z|[+-]\d{2}(:?\d{2})?)?$/
```
Assuming we can trust [Wikipedia](https://en.wikipedia.org/wiki/ISO_8601), it appears that both the colon and minutes offset are optional (but obviously not colon with no minutes).

Reply:
(Just noticed that Rubocop wants it frozen as well)

Reply:
They're not optional in [RFC3339](https://www.ietf.org/rfc/rfc3339.txt) (which is usually what people mean when they say ISO8601) so I'd be tempted to require them